app/server/apiGatewayDiscovery.ts

```diff
@@ -144,32 +142,26 @@ export const getAuthorisedExpressCallbackForApiGateway = (
       log.error(`Missing identity ID on the request object`);
       res.status(500).send();
     } else {
-      log.info(`proxying (${apiName}) : ${req.path}`);
-      fetch(
-        url.format({
-          protocol: "https",
-          host,
-          pathname: pathnameTransformer(req.params),
-          query: req.query
-        }),
-        {
-          method: req.method,
-          headers: {
-            "x-api-key": apiKey,
-            "x-identity-id": res.locals.identity.userId
-          },
-          body: req.method !== "GET" ? req.body : undefined
-        }
-      )
-        .then(intermediateResponse => {
-          res.status(intermediateResponse.status);
-          return intermediateResponse.json();
-        })
-        .then(responseBodyJSON => res.json(responseBodyJSON))
-        .catch(e => {
-          log.error(`error proxying (${apiName})`, e);
-          res.status(500).send();
-        });
+      const forwardQueryArgs = true;
+      return proxyApiHandler(`https://${host}`, {
+        "x-api-key": apiKey,
+        "x-identity-id": res.locals.identity && res.locals.identity.userId
+      })(straightThroughBodyHandler)(
+        path,
+        loggingCode,
+        forwardQueryArgs,
+        ...pathParamNamesToReplace
+      )(req, res);
     }
   };
```

Comment:
now re-uses `proxyApiHandler` for proxying the API Gateway Lambdas, for consistent logging and removing some code duplication